hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -85,7 +85,7 @@ def pulpcore_enabled?
     module_enabled?('foreman_proxy_content')
   end
 
-  def log_and_say(level, message, do_say = true, do_log = true)
+  def log_and_say(level, message, do_say: true, do_log: true)
```

Comment:
This is an API change and I don't see all invocations of `log_and_say` changed

Reply:
@ekohl it is changed now!

Reply:
@ekohl I think this changes still needs to be review?